<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QStackedWidget" name="MyWidget">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="search_line"/>
          </item>
          <item>
           <widget class="QPushButton" name="search_button">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Category:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Category">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>Food</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shopping</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Salary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Income</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Movies</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_2">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Transaction:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>Income</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Expense</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Filter">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Category</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Amount</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>Albert</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Food</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>2023-05-02</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>-780</string>
             </property>
            </item>
            <item row="0" column="4">
             <property name="text">
              <string>Expense</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>ElectroWorld</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>Shopping</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>2023-05-01</string>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string>-9990</string>
             </property>
            </item>
            <item row="1" column="4">
             <property name="text">
              <string>Expense</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>MyCompany</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>Salary</string>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string>2023-04-28</string>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string>19000</string>
             </property>
            </item>
            <item row="2" column="4">
             <property name="text">
              <string>Income</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>Maksym Yeromenko</string>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string>Income</string>
             </property>
            </item>
            <item row="3" column="2">
             <property name="text">
              <string>2023-04-26</string>
             </property>
            </item>
            <item row="3" column="3">
             <property name="text">
              <string>200</string>
             </property>
            </item>
            <item row="3" column="4">
             <property name="text">
              <string>Income</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>CineStar</string>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string>Movies</string>
             </property>
            </item>
            <item row="4" column="2">
             <property name="text">
              <string>2023-04-25</string>
             </property>
            </item>
            <item row="4" column="3">
             <property name="text">
              <string>-550</string>
             </property>
            </item>
            <item row="4" column="4">
             <property name="text">
              <string>Expense</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Account</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_1">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Surname</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_7">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_15">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Address</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_21">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Phone number</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_8">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>House number</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_16">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Date of birth</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QDateEdit" name="dateEdit_3">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_13">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Sex</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QRadioButton" name="radioButton_3">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Male</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QRadioButton" name="radioButton_4">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Female</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Country</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QComboBox" name="comboBox_5">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <item>
                <property name="text">
                 <string>Germany</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>USA</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Czech Republick</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ukraine</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Italy</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>England</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Spain</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QLineEdit" name="lineEdit_10">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_22">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="on_previousButton">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
               </property>
               <property name="text">
                <string>Transaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Save">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
               </property>
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Delete">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Bank</string>
    </property>
   </widget>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
